frontend/src/components/search/Search.tsx

```diff
@@ -2,29 +2,57 @@ import "./Search.scss";
 import axios, { CancelTokenSource } from "axios";
 import { useAlert } from "react-alert";
 import { useTranslation } from "react-i18next";
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import { axiosError } from "../../def/axios-error";
+import Modal from "react-responsive-modal";
+import { QuestionDef } from "../../def/QuestionDef";
+import Avatar from "../avatar/Avatar";
+import { formatDate } from "../../def/converter";
 
 let cancelToken: CancelTokenSource;
 
-export default function Search(props: { toggleSearch: () => void }) {
+/**
+ * Todo: Do
+ * @param props
+ * @constructor
+ */
+export default function Search(props: { isOpen: boolean, closeModal: () => void }) {
 	const alert = useAlert();
 	const { t } = useTranslation();
 	
-	const [suggestions, setSuggestions] = useState<any[]>([]);
+	const [suggestions, setSuggestions] = useState<QuestionDef[]>([]);
 	
 	const updateSuggestions = (query: string) => {
 		if (cancelToken) cancelToken.cancel("Operation canceled due to new request.");
 		cancelToken = axios.CancelToken.source();
 		
-		global.axios.get("question/search", {
+		global.axios.get<any>("question/search", {
 			params: {
 				q: query
 			},
 			cancelToken: cancelToken.token
 		})
 			  .then(res => {
-				  setSuggestions(res.data);
+				  let _suggestions: QuestionDef[] = [];
+				  res.data.forEach((_question: any) => _suggestions.push({
```

Comment:
Is this mock data or what is this object, with the hard coded values, representing?

Reply:
Yes, this is mock data because currently the backend does not provide the correct format.

Reply:
you can delete the mock data, the correct representation of the question is merged into main #261